pkg/utils/utils.go

```diff
@@ -272,6 +272,7 @@ func MonitorMounts() {
 			for _, vol := range csivolList.Items {
 				// ignore monitoring for volumes with deletion timestamp set
 				if vol.DeletionTimestamp != nil {
+					logrus.Infof("CVA: %v for volume: %v has been marked for deletion", vol.Name, vol.Spec.Volume.Name)
```

Comment:
How about below line?
```suggestion
					logrus.Infof("CVA: %v for volume: %v has been marked for cleanup activity", vol.Name, vol.Spec.Volume.Name)
```